File: src/main/java/com/vklochkov/model/Info.java
package com.vklochkov.model;

import javax.xml.bind.annotation.XmlRootElement;
import javax.xml.bind.annotation.XmlType;

@XmlRootElement(
    name = "Info"
)
@XmlType(
    propOrder = {
        "shopName", "foundationDate", "author", "about"
    }
)
public class Info {
    private String shopName;
    private String foundationDate;
    private String author;
    private String about;

    public Info() {}

    public Info(String shopName, String foundationDate, String author, String about) {
        this.shopName = shopName;
        this.foundationDate = foundationDate;
        this.author = author;
        this.about = about;
    }

    public String getShopName() {
        return shopName;
    }

    public void setShopName(String shopName) {
        this.shopName = shopName;
    }

    public String getFoundationDate() {
        return foundationDate;
    }

    public void setFoundationDate(String foundationDate) {
        this.foundationDate = foundationDate;
    }

    public String getAuthor() {
        return author;
    }

    public void setAuthor(String author) {
        this.author = author;
    }

    public String getAbout() {
        return about;
    }

    public void setAbout(String about) {
        this.about = about;
    }

    @Override
    public String toString() {
        return "Info{" +
            "shopName='" + shopName + '\'' +
            ", foundationDate='" + foundationDate + '\'' +
            ", author='" + author + '\'' +
            ", about='" + about + '\'' +
            '}';
    }
}

File: client/src/app/model/Info.ts
export class Info {
  private _about: String;
  private _author: String;
  private _foundationDate: String;
  private _shopName: String;

  constructor (about: String, author: String, foundationDate: String, shopName: String) {
    this._about = about;
    this._author = author;
    this._foundationDate = foundationDate;
    this._shopName = shopName;
  }

  get about(): String {
    return this._about;
  }

  get author(): String {
    return this._author;
  }

  get foundationDate(): String {
    return this._foundationDate;
  }

  get shopName(): String {
    return this._shopName;
  }
}
